src/sentry/workflow_engine/endpoints/organization_data_condition_index.py

```diff
@@ -0,0 +1,88 @@
+from typing import NotRequired, TypedDict
+
+from drf_spectacular.utils import extend_schema
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.workflow_engine.endpoints.serializers import DataConditionHandlerSerializer
+from sentry.workflow_engine.models.data_condition import IGNORED_CONDITIONS
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler
+
+
+class DataConditionRequestSerializer(serializers.Serializer):
+    type = serializers.ChoiceField(
+        choices=[type.value for type in DataConditionHandler.Type],
+    )
```

Comment:
🤔 i think we should probably move this to a validator -- the design would be that a "serializer" is for any outgoing data and a validator is for any incoming data.

By splitting this off and into that directory we can reuse it for any api that may need this info.

---

Alternatively, this is also pretty small, and we could just do this inline too:
```
handler_group = request.GET.get("group")
try:
    DataConditionHandler.Type(handler_group)
except:
    ValidationError("Invalid group")
```
(PS, i swapped in `group` for `type` based on the other comment about the result)

Reply:
Doing this inline makes sense to me!